src/main/java/karuna/karuna_backend/security/CustomAuthenticationManager.java

```diff
@@ -1,4 +1,4 @@
-package karuna.karuna_backend.Authentication;
+package karuna.karuna_backend.security;
```

Comment:
It feels like it belongs more to security -> config package instead of plain security one.